### PR TITLE
Add easy build script for testing local changes

### PR DESCRIPTION
**Pull request by ajcoppa**
I made this mini script to make it easy to test local changes while keeping them separate from the main TourGuide install. Let me know if you think this would be helpful.

### CODE REVIEW

File: test_local_changes.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+
+import Bundle_ASH_script
+
+# If needed, replace this with the path to your mafia's base config folder
+MAFIA_DIRECTORY = f"/Users/{os.getlogin()}/Library/Application Support/Kolmafia"
```
Comment (ajcoppa) [inline test_local_changes.py:9]: Not sure if Mafia has an easy default to add here for Windows, but happy to add it (and possibly OS-detection too?) if so.

File: test_local_changes.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (docrostov) [reply test_local_changes.py:1]: i think it's fine to expect, sure. i know i have it. i don't think TES does? but even if TES doesn't he has his own testing workflow lol.

File: test_local_changes.py (hunk 1 above)
Comment (docrostov) [reply test_local_changes.py:9]: it does not unfortunately. i think maybe we'll want to make the test_local_changes.py file have an argument on run that references a directory. pretty sure this definitely isn't where mine is though i'll check later. i am on windows so i can be a guinea pig here :-)

File: test_local_changes.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+if not os.path.exists(MAFIA_DIRECTORY):
+    print(f"""Path {MAFIA_DIRECTORY} doesn't exist.
+Try editing MAFIA_DIRECTORY in {os.path.basename(__file__)} to your mafia's base config folder.
+""")
+    exit(1)
+
+if os.path.exists(BUNDLED_SCRIPT_NAME):
+    os.remove(BUNDLED_SCRIPT_NAME)
+
+Bundle_ASH_script.bundle_and_write("relay/relay_TourGuide.ash", BUNDLED_SCRIPT_NAME, "Source")
```
Comment (docrostov) [inline test_local_changes.py:23]: one quick question on this. any reason we aren't doing a TS style bundle like what gets run when it gets packaged to release? i am actually (largely) a python dev so this workflow makes sense to me but i think we already have a bundler, exposing the one we already have might be better than this even.

**Review by docrostov — COMMENTED** [review]
not approving yet, want to test it. but thank you for pushing this up. i'll try to test it tomorrow in between work!

File: test_local_changes.py (hunk 3 above)
Comment (ajcoppa) [reply test_local_changes.py:23]: No real reason, I'd actually prefer something more TS-style if there's a good example of that! I just modeled this after what the GitHub action is doing. https://github.com/loathers/TourGuide/blob/main/.github/workflows/main.yml#L317

**Comment by ajcoppa** [comment]
Going to close this for cleanup's sake, happy to put up again later if desired

**Pull request closed without merging**